src/modules/acc/acc_cdr.c

```diff
@@ -51,18 +51,18 @@
 
 /* Solaris does not provide timersub macro in <sys/time.h> */
 #ifdef __OS_solaris
-#define timersub(tvp, uvp, vvp)                     \
```

Comment:
Are lines 54 to 61 shown in the diff just because of different whitespaces, without any code change?

Reply:
I don't understand, I didn't change them, must be something from my editor.

Reply:
fixed

Reply:
+1